app/mailers/visitor_mailer.rb

```diff
@@ -15,12 +15,16 @@ def request_acknowledged(visit)
 
   def booked(visit)
     I18n.locale = visit.locale
+    @message = visit.acceptance_message
```

Comment:
For example if the acceptance was a first-class event, it could be passed here – rather than the visit, which does a rather horrible lookup of acceptance message via its state change.